import_map.json

```diff
@@ -4,6 +4,7 @@
     "std/": "https://deno.land/std@0.170.0/",
     "$fresh/": "https://deno.land/x/fresh@1.1.2/",
     "preact-render-to-string": "https://esm.sh/*preact-render-to-string@5.2.4?pin=102",
+    "json-schema": "https://esm.sh/@types/json-schema@7.0.11?pin=102",
```

Comment:
I think we started using `deps.ts` file instead of adding it to the manifest. IIRC, this will require the site to include the `json-schema` entry in the manifest

Reply:
>One thing that comes to mind is: what do you think about adding a verbose or debug flag that outputs all steps performed in the resolution? If god forbid I'd have to fix a bug here, the first thing I'd do would be set up something like that to understand what's going on.

Definitely! I think we can add this in the upcoming PRs, WDYT?

> I think we started using deps.ts file instead of adding it to the manifest. IIRC, this will require the site to include the json-schema entry in the manifest

Will do, thanks!